### DISCUSSION TITLE
Change default port (443)

### DISCUSSION THREAD

**Discussion by Phelecks** [hiddify/Hiddify-Manager#530] [Q&A سوالات رایج]
How to change default port (443) on installation step? چگونه پورت 443 را موقع نصب تغییر بدم.
Now I have services listening on this port and cant change it.

**Comment by MasoudRahmani** [comment]
They are using customized nginx, so in order to change any configuration for incoming traffic you need to look through `cd /opt/hiddify-config/nginx` 

In this folder you see `parts` and `config.d` folder plus `nginx.conf` file... 

you need to have a good understanding of nginx since configs are separated and dependent on each other. 

be carefull.

**I wish they'd either keep nginx folder and file structure or just use default nginx.**
